content/chainguard/chainguard-images/working-with-images/cve_visualizations/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Using CVE Visualizations"
+linktitle: "CVE Visualizations"
+type: "article"
+description: "Getting started with the CVE Visualization feature."
+date: 2024-12-04T11:07:52+02:00
+lastmod: 2024-12-04T11:07:52+02:00
+draft: false
+tags: ["CONCEPTUAL", "CHAINGUARD IMAGES", "PRODUCT"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 020
+toc: true
+---
+
+The CVE Visualizations feature in the Chainguard Console helps teams measure the engineering,
```

Comment:
Maybe we can open or add this positioning statement somewhere: "CVE Visualizations are a single source of truth that bring transparency to image health and continuously articulate the engineering, security, and economic benefits of adopting Chainguard's guarded, minimal, zero-CVE images to your internal stakeholders."

Reply:
I reworded a bit, but these docs are developer facing. We try to keep the language concise and focused on getting tasks done, whereas the language here is more appropriate for a blog or marketing flyer.